### PR TITLE
(OLD BRANCH) Job teaser redesign and some other fixes

### PR DESCRIPTION
**Pull request by kernazhytskayalena**

### CODE REVIEW

File: src/styles/includes/_job-teaser.scss [hunk 1]
```diff
@@ -56,4 +171,30 @@
...
+	.switch {
+		height: 5.5rem;
+		margin-bottom: -1.3rem;
+
+		@include bp-xs {
+			height: 3.9rem;
+		}
+
+		@include bp-s {
+			height: 4.6rem;
```
Comment (moritzbehnke) [inline src/styles/includes/_job-teaser.scss:184]: Font is a tiny bit too low

File: src/styles/includes/_job-teaser.scss [hunk 2]
```diff
@@ -56,4 +171,30 @@
 			}
 		}
 	}
+
+	.switch {
+		height: 5.5rem;
+		margin-bottom: -1.3rem;
+
+		@include bp-xs {
+			height: 3.9rem;
```
Comment (moritzbehnke) [inline src/styles/includes/_job-teaser.scss:180]: Font is noticeably too low

File: src/styles/includes/_job-teaser.scss [hunk 3]
```diff
@@ -56,4 +171,30 @@
...
+		}
+
+		@include bp-s {
+			height: 4.6rem;
+		}
+
+		@include bp-m {
+			height: 4.5rem;
```
Comment (moritzbehnke) [inline src/styles/includes/_job-teaser.scss:188]: Font is a tiny bit too low

File: src/_data/landingpage.json [hunk 4]
```diff
@@ -235,6 +235,24 @@
 		"button": {
 			"href": "https://sinnerschrader.jobs",
 			"text": "Join us"
+		},
+		"images": {
+			"logoPart1": {
```
Comment (moritzbehnke) [inline src/_data/landingpage.json:240]: How about namig the keys and images a little bit more clearer? For this one maybe `logoLeftHalf`.

File: src/_data/landingpage.json [hunk 5]
```diff
@@ -235,6 +235,24 @@
...
+				"src": "/assets/svg/logo-part.svg",
+				"alt": ""
+			},
+			"logoPart2": {
```
Comment (moritzbehnke) [inline src/_data/landingpage.json:244]: For this one maybe `logoRightHalf`

File: src/_data/landingpage.json [hunk 6]
```diff
@@ -235,6 +235,24 @@
...
+				"src": "/assets/svg/logo-part2.svg",
+				"alt": ""
+			},
+			"element1": {
+				"src": "/assets/svg/element1.svg",
```
Comment (moritzbehnke) [inline src/_data/landingpage.json:249]: For this one maybe `ovalShape`

File: src/_data/landingpage.json [hunk 7]
```diff
@@ -235,6 +235,24 @@
...
+				"alt": ""
+			},
+			"element2": {
```
Comment (moritzbehnke) [inline src/_data/landingpage.json:252]: For this one maybe `circleShape`

**Review by moritzbehnke — CHANGES_REQUESTED** [review]
Also please rebase to the latest development

**Comment by wheelmaker24** [comment]
I've tried to rebase this branch onto the current development branch, but failed. Any idea on how to proceed with this branch? @moritzbehnke ? ;)

**Comment by evgenypastushek** [comment]
@wheelmaker24 
here is new PR https://github.com/sinnerschrader/sinnerschrader-reboot/pull/747
this one can be closed

**Pull request closed without merging**